app/controllers/claims_form_callbacks.rb

```diff
@@ -84,7 +84,7 @@ def on_school_search_results?
   end
 
   def no_eligible_itt_subject?
-    !current_claim.eligible_itt_subject
+    !journey_session.answers.eligible_itt_subject && !journey_session.answers.early_career_payments_dqt_teacher_record&.eligible_itt_subject_for_claim
```

Comment:
How come we need to check the dqt_teacher_record here?
I thought we'd only need to access the dqt teacher record when setting the attributes on the answers and otherwise we'd just check the answers for the eligible_itt_subject.
Does the test suite pass if we remove the right hand side of the `&&`?

Reply:
it does seem to pass so totally removing it. it'll be from my many attempts at getting things to pass